trough/client.py

```diff
@@ -175,7 +177,7 @@ def read_urls_for_regex(self, regex):
         `{segment: url}`
         '''
         d = {}
-        reql = self.rr.table('services', read_mode='outdated')\
+        reql = self.rr.table('services', read_mode = 'outdated')\
```

Comment:
The cleanup here should be to replace the backslashes with () as discussed in PEP8.